src/Particles.jl

```diff
@@ -26,17 +29,20 @@ end
     ChargeInv = 2
 end
 
-struct MeasuredValue{T}
-    value::T
-    lower_limit::T
-    upper_limit::T
+struct MeasuredValue{D} 
+    value::Quantity{T1,D,U1} where {T1 <: Real, U1 <: Unitful.Units}
```

Comment:
Ah I see, the limits can have different unit-prefixes? Is it guaranteed that we do not mix `cm` with `kg`? ;-)